src/test/java/com/microsoft/sqlserver/jdbc/fips/FipsTest.java

```diff
@@ -52,8 +52,7 @@ public void fipsTrustServerCertificateTest() throws Exception {
         }
         catch (SQLServerException e) {
             Assertions.assertTrue(
-                    e.getMessage().contains("Could not enable FIPS due to either encrypt is not true or using trusted certificate settings."),
-                    "Should create exception for invalid TrustServerCertificate value");
+                    e.getMessage().contains("Unable to verify FIPS mode settings."));
```

Comment:
Any specific reason for removing `"Should create exception for invalid TrustServerCertificate value"` from line 56?